devops/scripts/benchmarks/benches/compute.py

```diff
@@ -264,6 +264,24 @@ def parse_unit_type(compute_unit):
 
 
 class ComputeBenchmark(Benchmark):
+
+    # List of benchmarks that should not be traced by Unitrace
```

Comment:
if there's some issue with SubmitGraph under unitrace, I'd prefer if you just override the traceable function in that benchmark's class.